src/constants/colors/primary.ts

```diff
@@ -0,0 +1,60 @@
+import PropTypes from 'prop-types';
+
+import COLORS from '.';
```

Comment:
`src/constants/colors/index.ts` is unchanged to keep the diff to a minimum, but introducing full theming support would require making changes to that file as well. We'd want to disallow access to `COLORS` directly except under specific circumstances. 

I think our goal should be to make opting-in to the Theme as easy as possible. Personally I've already enjoyed not needing to import `COLORS` and just access it (and already typed!) off of a theme.